### PR TITLE
Bell button notifications layout

### PR DESCRIPTION
**Pull request by INTJT**

### CODE REVIEW

**Comment by INTJT** [comment]
[Bell button notifications layout](https://trello.com/c/jVcmMbcm/123-bell-button-notifications-layout)

File: client/package.json [hunk 1]
```diff
@@ -27,11 +27,13 @@
 		"clsx": "^1.1.1",
 		"formik": "^2.2.9",
 		"jwt-decode": "^3.1.2",
+		"moment": "^2.29.1",
```
Comment (svitlua) [inline client/package.json:30]: moment is a very heavy library
it is usually recommended to use something lighter, like dayjs (which is super light) or similar
but i think for our project moment is fine

File: client/src/components/common/notification/helper.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { ROUTES } from 'constants/routes';
+import { NotificationTypes, TNotification } from 'typings/common/INotification';
+import { Rank } from 'components/basic';
+import { INotificationProps } from './';
+
+export function mapNotificationToProps(notification: TNotification): INotificationProps {
+	const userLink = ROUTES.UserProfile + '/id';
```
Comment (svitlua) [inline client/src/components/common/notification/helper.tsx:8]: We will use users/**nickname** for user links, **not id**

File: client/src/containers/header/index.tsx [hunk 3]
```diff
@@ -11,52 +12,55 @@ import menuSignout from 'assets/icons/header/menu/logout.svg';
...
-
-
 const Header = () => {
-    const history = useHistory();
-    const dispatch = useDispatch();
-    const handleLogout = async () => {
-        dispatch(actions.logoutUser());
-    };
-    const { user } = useAppSelector((state) => state.auth.userData);
+	const history = useHistory();
```
Comment (svitlua) [inline client/src/containers/header/index.tsx:16]: there is a historyHelper that you could use:
`import historyHelper from 'helpers/history.helper';`

**Review by svitlua — APPROVED** [review]
There ar some small things that can be improved, I have left comments, but nothing critical

**Pull request merged**